src/schema/rules/sidecars/func.yaml

```diff
@@ -13,6 +13,9 @@ MRIFuncRequired:
     selectors:
     - modality == "mri"
     - datatype == "func"
+    - '!("RepetitionTime" in sidecar)'
+    - '!("VolumeTiming" in sidecar)'
```

Comment:
It looks like this requires TR and VolumeTiming only if those fields aren't present? I don't think these two new selectors should be included.

Reply:
My thinking was that if neither of them are actually there we make them required in order to trip a validation error and cause validation to fail. it might be too clever by half. I'll think of another way to do this or a better way to document it.

Reply:
As discussed earlier today, I think we want to split this into two tables. Each could be something like:

```yaml
MRIFuncRepetitionTime:
    selectors:
    - modality == "mri"
    - datatype == "func"
    - '!("VolumeTiming" in sidecar)'
    - '[".nii", ".nii.gz"].includes(extension)'
    fields:
        RepetitionTime:
            level: required
            level_addendum: mutually exclusive with `VolumeTiming`

MRIFuncVolumeTiming:
    selectors:
    - modality == "mri"
    - datatype == "func"
    - '!("RepetitionTime" in sidecar)'
    - '[".nii", ".nii.gz"].includes(extension)'
    fields:
        VolumeTiming:
            level: required
            level_addendum: mutually exclusive with `RepetitionTime`
```